styles.css

```diff
@@ -0,0 +1,960 @@
+}
+
+.intro2 {
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  align-items: flex-start;
+  padding-left: 24px;
+}
+
+.social {
+  display: flex;
+  list-style: none;
+}
+
+ul.social {
+  -webkit-margin-before: 1px;
+  margin-block-start: 1px;
+  -webkit-margin-after: 1px;
+  margin-block-end: 1px;
+  -webkit-padding-start: 0;
+  padding-inline-start: 0;
+}
+
+.social-icon {
+  align-items: center;
+  justify-content: center;
+  height: 20px;
+  padding: 10px 0 0 20px;
+  transition: all 2s;
+}
+
+.social-icon:hover {
+  transform: rotate(360deg);
+}
+
+li.social-icon {
+  display: flex;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  width: 90%;
+  height: 600px;
+  left: calc(50% - 327px / 2);
+  top: 114px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 8%;
+}
+
+.title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding-top: 10px;
+  padding-bottom: 20px;
+}
+
+.role {
+  display: grid;
+  grid-template-columns: 1fr 0.5fr 1.5fr 0.5fr 1.5fr 0fr;
+}
+
+.role-name {
+  position: static;
+  width: 55px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 2px;
+}
+
+.role-dot {
+  width: 10px;
+  height: 10px;
+  left: 67px;
+  top: 12px;
+  background: #c1c7d0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 2.5px 12px;
+  border-radius: 50%;
+}
+
+.role-job {
+  position: static;
+  width: 88px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.role-year {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.project-description {
+  margin-left: 5%;
+}
+
+.project-description > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.project-details {
+  width: 70%;
+  height: 76px;
+  left: 0;
+  top: 92px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer, notice the space between each element. Kindly compare your code and ensure it matches the figma design as much as possible. You can fix that by adding `margin-top` and `margin-bottom` CSS property to the `.project-details` class.